nucleus/common/common-util/src/test/java/com/sun/enterprise/universal/process/ProcessManagerTest.java

```diff
@@ -132,11 +131,12 @@ void testWaitUntilTextInStdErr() {
     @Timeout(value = 5, unit = TimeUnit.SECONDS)
     @DisabledOnOs(WINDOWS)
     void testDetectTextInStdOutIfProcessStops() throws InterruptedException {
+        int sleepTimeSeconds = 1;
         ProcessManager pm = new ProcessManager("sh", "-c", "echo \"start\nhello\ncontinue\"");
         pm.setEcho(false);
         pm.setTextToWaitFor("hello");
         int exitCode = assertDoesNotThrow(pm::execute);
-        Thread.sleep(ofSeconds(1));
+        Thread.sleep(sleepTimeSeconds * 1000);
```

Comment:
You can write also 1_000L, no need to create a variable if it doesn't have any other usage.

Reply:
I wanted to do that first, but then I saw that other test method follow this pattern so I did it this way. If we used Java 21, I would rather use Duration, but with millis I’m not sure which approach is better.